scripts/minikube.sh

```diff
@@ -50,16 +50,28 @@ function detect_minikube() {
 
 # install minikube
 function install_minikube() {
+     local mku_version latest_version
+     mku_version=$(${minikube} update-check 2> /dev/null || true)
+     latest_version=$(echo ${mku_version} | cut -d' ' -f4)
+     if [[ -z "${latest_version}" ]]; then
+         # skip: update-check failed for some reason, lets continue with what we have
+         latest_version=${MINIKUBE_VERSION}
+     fi
     if type "${minikube}" >/dev/null 2>&1; then
         local mk_version version
         read -ra mk_version <<<"$(${minikube} version)"
         version=${mk_version[2]}
         echo "minikube already installed with ${version}"
-        return
+        MINIKUBE_VERSION=${version}
+        if [[ "${version}" == "${latest_version}" ]]; then
+          echo "minikube is already the latest version"
```

Comment:
weird indention, make it 4 spaces please.

Reply:
tweaked my vimrc now, thanks!